src/Widgets/AppContextMenu.vala

```
@@ -108,6 +108,14 @@ public class Slingshot.AppContextMenu : Gtk.Menu {
         });
     }
 
+    private void open_in_appcenter () {
+        try {
+            Process.spawn_command_line_async ("xdg-open appstream://" + appstream_comp_id);
+        } catch (SpawnError e) {
+            debug ("Unable to open in Appcenter: %s\n", e.message);
```

I think we probably want this to throw a dialog and not just be a silent fail